### PR TITLE
fix: add missing APAC region prefix in Bedrock models

### PR DESCRIPTION
**Pull request by christian-bromann**
## Add support for APAC region prefix in Bedrock models

- Add "apac" to AWS_REGIONS list in both chat models and LLMs
- Add comprehensive test suite for model provider detection
- Test various regional model prefixes (eu, apac, us-gov, us)
- Ensure proper handling of different model ID formats and providers
- Add validation for unsupported custom model ARNs

This enables support for APAC-specific Bedrock model IDs like: `apac.anthropic.claude-3-5-sonnet-20240620-v1:0`

This change resolves the issue where users in APAC regions couldn't use region-specific Bedrock model identifiers. Previously, attempting to use models with the `apac.` prefix would result in an "Unknown model provider" error because the APAC region prefix wasn't recognized in the AWS_REGIONS configuration.

The fix includes:
- Adding "apac" to the supported AWS regions in both `BedrockChat` and `Bedrock` LLM classes
- Comprehensive test coverage for all regional model prefixes including edge cases
- Validation that ensures proper error handling for unsupported custom model ARNs

Fixes [#8408](https://github.com/langchain-ai/langchainjs/issues/8408)

### CODE REVIEW

File: libs/langchain-community/src/llms/bedrock/web.ts [hunk 1]
```diff
@@ -28,6 +31,7 @@ const AWS_REGIONS = [
   "ap",
   "af",
   "us-gov",
+  "apac",
```
Comment (hntrl) [inline libs/langchain-community/src/llms/bedrock/web.ts:35]: Could we also add mexico (`mx`)?
```suggestion
  "apac",
  "mx",
```

Comment (christian-bromann) [reply libs/langchain-community/src/llms/bedrock/web.ts:35]: Done!

**Review by hntrl — APPROVED** [review]
